packages/calcite-components/src/demos/sortable-list.html

```diff
@@ -152,15 +152,7 @@ <h1 style="margin: 0 auto; text-align: center">Sortable list</h1>
             <calcite-block drag-handle="true" heading="Block A" collapsible="" calcite-hydrated="" open>
               <calcite-label alignment="start" scale="m" layout="default" calcite-hydrated="">content A </calcite-label>
             </calcite-block>
-            <calcite-block
-              drag-handle="true"
-              heading="Block B"
-              intl-collapse="Collapse"
-              intl-expand="Expand"
```

Comment:
Unrelated change: These props are no longer supported.

Reply:
FIXME: since it’s unrelated, can you submit it in a separate PR? It’s isolated, so if you submit and merge that change to `dev`, then sync this PR’s branch, it’ll clean things up without needing to remove anything here.